### PR TITLE
Async stats computation

### PR DESCRIPTION
**Pull request by siddg97**
- Updated swagger docs
- Added celery worker logs
- Task can now be triggered upon submitting an answer to a question
- A task status can be obtained by polling at the task status endpoint
- Endpoints to fetch results for all questions belonging to the user and fetching results for only a specific user

### CODE REVIEW

File: server/app/routes/results.py [hunk 1]
```diff
@@ -32,54 +28,35 @@ def index(question_id):
...
-    # for each in user["bucket_files"]:
-    #     file_path = f"{bucket_dir}/{each}"
-    #     transcript = get_transcript(f"gs://{GCS_BUCKET}/{file_path}")
-    #     stats = get_stats(transcript, each, TMP_DIR)
-    #     all_stats.append(stats)
+    @app.route("/api/questions/results", methods=["GET"])
+    def get_results_user():
+        user_id = get_user_cookie()
+        if not user_id:
+            raise ForbiddenError()
```
Comment (jacob-pauls) [inline server/app/routes/results.py:38]: I've seen this logic repeated across multiple endpoints. Its simple, but would we benefit by extracting this to some sort of exception handling file that checks common exceptions? We could pass the user_id each time, just to follow the DRY principle.

Also see lines 13-16 of this file, lines 9-11 of `task_status.py`. etc.

**Review by jacob-pauls — COMMENTED** [review]
Looks great! Feel free to merge if you think extracting the logic on those exceptions is unnecessary.

File: server/app/routes/results.py (hunk 1 above)
Comment (siddg97) [reply server/app/routes/results.py:38]: Good catch. What do you think about the `ForbiddenError` exception being raised inside `get_user_cookie()? That way we can simply use `get_user_cookie()` for any existing future routes. If no cookie is found, the said exception is raised and handled by the error handler

Comment (jacob-pauls) [reply server/app/routes/results.py:38]: Exactly! Even better idea. That would be perfect.

**Review by shirleyvong — APPROVED** [review]
LGTM

**Review by shirleyvong — DISMISSED** [review]

**Review by shirleyvong — APPROVED** [review]

**Pull request merged**